packages/nextra/src/loader.js

```diff
@@ -21,6 +21,17 @@ function getFileName (resourcePath) {
   return removeExtension(path.basename(resourcePath))
 }
 
+const parseJsonFile = (content, path) => {
+  let parsed
```

Comment:
```suggestion
  let parsed = {}
```

So even if it fails, the site can still be built.

Reply:
Wanted to do that, good catch.